resources/blueprints/collections/brands/brand.yaml

```diff
@@ -24,6 +24,51 @@ sections:
           instructions_position: above
           visibility: visible
           localizable: true
+      - handle: description
```

Comment:
Better use the fieldset: https://github.com/rapidez/statamic/blob/master/resources/fieldsets/content.yaml, this way the configuration is in 1 place.

Reply:
That's not possible as the pagebuilder is in this. The content fieldset and the pagebuilder have the same handle so they would overwrite eachother.

Reply:
If I remember correctly; it's possible to give it another handle